builds/Makefile

```diff
@@ -2075,10 +2075,10 @@ TV_CROSS_CONFIGURE_FLAGS=$(CROSS_BASE_CONFIGURE_FLAGS)	\
 	--prefix=$(BUILD_DESTDIR)/crosstv \
 	--cache-file=../crosstv.config.cache \
 	--target=aarch64-unknown-darwin \
+	--with-llvm=../llvm64/usr
 
 TV_CROSS_CONFIGURE_ENVIRONMENT = \
-	$(Q) PATH="../llvm64/usr/bin:$(PATH)" \
-	CFLAGS="$(TV_CROSS_CFLAGS)" \
```

Comment:
Did you intent to remove `CFLAGS` here as well (you didn't remove it for the other platforms)?